src/main/java/core/basesyntax/service/impl/CsvFileTransactionReaderService.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.TransactionReaderService;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvFileTransactionReaderService implements TransactionReaderService {
```

Comment:
This class doesn't have any specific logic to transactions or csv, it just reads lines from file
If you pass to it file with any other information or in different format it would produce result
```suggestion
public class FileReaderImpl implements FileReader {
```